Biasing/src/Biasing/DeepEcalProcessFilter.cxx

```diff
@@ -52,17 +53,22 @@ void DeepEcalProcessFilter::stepping(const G4Step* step) {
   }
 
   // Check in which volume the particle is currently
-  auto volume{track->GetVolume()->GetLogicalVolume()
-                  ? track->GetVolume()->GetLogicalVolume()->GetName()
-                  : "undefined"};
+  auto volume = track->GetVolume()->GetLogicalVolume();
+  auto volume_name = volume->GetName();
+  if (!volume_name) {
+    volume_name = "undefined";
+  }
+  // auto volume{track->GetVolume()->GetLogicalVolume()
+  //                 ? track->GetVolume()->GetLogicalVolume()->GetName()
+  //                 : "undefined"};
```

Comment:
Delete these, although I'm not sure what the advantage of your new lines are wrt to what is now to be deleted